source/_integrations/broadlink.markdown

```diff
@@ -33,6 +33,7 @@ remote:
   - platform: broadlink
     host: IP_ADDRESS
     mac: MAC_ADDRESS
+    type: 0x5f36
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
Done.